doc/project/checklist.md

```diff
@@ -491,7 +491,7 @@ For a transition from S0 to S1, the following items are expected be completed.
 
 ### DIF_USED_IN_TREE
 
-All existing in-tree code which uses the device, uses the device via the DIF.
+All existing non-production in-tree code which uses the device, uses the device via the DIF.
 There is no remaining driver code that directly uses the device outside of DIF code.
```

Comment:
I think delete this sentence, seems redundant and slightly misleading since drivers are likely to be production code.